internal/render/node.go

```diff
@@ -35,6 +35,7 @@ func (Node) Header(_ string) Header {
 		HeaderColumn{Name: "NAME"},
 		HeaderColumn{Name: "STATUS"},
 		HeaderColumn{Name: "ROLE"},
+		HeaderColumn{Name: "ARCH"},
```

Comment:
I don't think we should volunteer this info all the time. Perhaps add a wide option?

Reply:
Yes, agree on that. I'll update accordingly.